### ISSUE TITLE
Appium inspector unable to inspect elements in ios application

### ISSUE THREAD

**Issue by SangramMedic** [appium/appium#12023]
I am using appium 1.9.0. Appium inspector unable to inspect elements in ios application. It was working fine in 1.7.2. Can you please review the attached screenshot.
![screenshot 2019-01-11 at 10 23 27 am](https://user-images.githubusercontent.com/46773852/51299901-f401ad80-1a4f-11e9-99cd-ba15033f46ed.png)

**Comment by mykola-mokhnach** [comment]
- Please provide proper issue description
- Appium 1.9.0 is not the most recent one
- If this is a web view then it is recommended to use hybrid testing approach

**Comment by SangramMedic** [comment]
We have checked this on Appium 1.10.0 as well. But we are getting same issue.

**Comment by mykola-mokhnach** [comment]
Is the application accessible? Do you see any elements if it is opened in the Accessibility Inspector?

**Comment by DGandalf** [comment]
The same issue can be seen in the iOS Settings of a real device.
If you move the mouse pointer to the "<" sign in the top left screen area the whole screen gets blue aka selected instead the "<" element

If you use the inspector tree and search for that "<" element and if you then select this element in the inspector element tree the "<" element will the selected in the right manner.

I'm using Appium 1.10.0.

**Comment by mykola-mokhnach** [comment]
Again, can you see the accessibility elements in the Xcode's accessibility inspector?

**Comment by SangramMedic** [comment]
Yes, I am able to see the accessibility elements in the Xcode's accessibility inspector.
Please check the attached screenshot.

<img width="548" alt="screenshot 2019-01-18 at 9 10 29 am" src="https://user-images.githubusercontent.com/46773852/51364216-68485980-1b01-11e9-9266-2e540b134555.png">

**Comment by ericpolkadoc** [comment]
Hi,

I am seeing this same issue.  However in my view, I'm seeing much more of the element tree.  

Has there been any more reports of this issue?  

Im using Appium 1.13.0.  The row element children under the highlighted wrapper are not listed in the element tree.  If I use XCode Accessibility Inspector, I have no tree when clicking the arrow next to the name of the parent view in the Hierarchy section

![image](https://user-images.githubusercontent.com/13222265/62743930-3f35da80-b9f9-11e9-8546-eb9ab0916250.png)

![image](https://user-images.githubusercontent.com/13222265/62744004-845a0c80-b9f9-11e9-8fba-05dc9a645f3d.png)

**Comment by mykola-mokhnach** [comment]
Can be related to https://github.com/appium/appium/issues/6517

**Comment by ericpolkadoc** [comment]
do we know if there is anything we can do about it?  Where do you feel that the issue is stemming from?
